<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="pb_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pb_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pb_point">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pb_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pb_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pb_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pb_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pb_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pb_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pb_rem">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pb_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pb_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pb_back_l">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pb_back_r">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pb_enter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pb_umn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pb_minus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pb_del">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pb_plus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pb_clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="ch_Box_1">
       <property name="text">
        <string>Inline string saving</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ch_Box_clean">
       <property name="text">
        <string>Full clean</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="ch_Box_oneline">
       <property name="text">
        <string>Oneline exp</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_0</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_5</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_4</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_1</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_point</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_back_r</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_back_l</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_9</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_8</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_7</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_6</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_del</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_umn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_plus</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_minus</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ins_sym()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ins_sym()</slot>
 </slots>
</ui>
